content/aips/MAIPolyV3CapUpdate.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Update MAI Caps Aave Polygon V3 pool
+author: Marc Zeller (@marczeller), Aave-Chan initiative
+shortDescription: Update MAI Caps Aave Polygon V3 pool
+discussions: https://governance.aave.com/t/arfc-increase-borrow-cap-for-mai-aave-polygon-v3/11547
+created: 2023-02-16
+updated: 2023-02-16
+---
+
+## Simple Summary
+This AIP proposal presents Aave with the opportunity to update MAI supply & borrow caps on the Polygon v3 Liquidity Pool.
+
+## Motivation
+
+Supporting stablecoin diversity is part of the Aave-Chan Initiative (ACI) delegate platform.
+
+The ACI does not have any link nor is paid by QiDAO or any entity to present this AIP.
+
+Supply and borrow caps are suggested for this asset to limit risk exposure, the caps have been proposed by a join collaboration between Gauntlet & Chaos Labs.
+
+## Specification
+
+Ticker: MiMATIC
+
+Contract Address: [0xa3Fa99A148fA48D14Ed51d610c367C61876997F1](https://polygonscan.com/address/0xa3Fa99A148fA48D14Ed51d610c367C61876997F1)
+
+|Risk Parameter|Value|
+| --- | --- |
+|Supply Cap|1,100K|
+|Borrow Cap|600k|
+
+
+The proposal payload uses the [GenericListingEngine](https://etherscan.io/address/0xC51e6E38d406F98049622Ca54a6096a23826B426#code) to perform a new asset listing with the following parameters:
+
+```solidity
+IProposalGenericExecutor {
```

Comment:
```suggestion
contract MAIV3PolCapsPayload is IProposalGenericExecutor {
```

this section is a bit weird - seems like you just copied the code, which is guess is actually fine as it's simple enough, but you copied only part, so now it's not valid code.